src/modules/vtol_att_control/tiltrotor.h

```diff
@@ -99,8 +99,7 @@ class Tiltrotor : public VtolType
 	float _tilt_control;
 	float _roll_weight_mc;
```

Comment:
Nitpick: this param should probably be called _weight_mc because it has nothing to do with roll, right?